e2e/framework/helper.go

```diff
@@ -29,6 +29,35 @@ import (
 	empty "google.golang.org/protobuf/types/known/emptypb"
 )
 
+var (
+	DefaultTimeout = time.Second * 10
```

Comment:
why 10 seconds to finish the test?

Reply:
It's not used for tests, but for node joins:
https://github.com/0xPolygon/polygon-edge/blob/3cd29198a1f78e51e40ed333013202c3bf75df45/e2e/framework/helper.go#L248

Reply:
UPDATE:

I've swapped out all occurrences of the `time.second * 10` for the `DefaultTimeout`, check out a comment that's further down